### DISCUSSION TITLE
Enable sort by Assignee

### DISCUSSION THREAD

**Discussion by JulieLeeMSFT** [community/community#45831] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

Please enable sort by Assignee in GitHub Issues and Pull Requests.

**Comment by minware-dan** [comment]
Hey @JulieLeeMSFT - this is something that we handle really well in minware. Is your request majorly disruptive to your workflow or a minor inconvenience?

**Reply by JulieLeeMSFT** [reply]
Sorry, but what is minware? I thought that I was requesting a feature for GitHub. I maintain dotnet/runtime area-codegen-coreclr. And, there are more than 1000 issues that I need to triage and manage. So, I need a feature to sort them by assignee.  
I don't see sort by Assignee, sort by Author or sort by Milestone and so on. Out of them, I need sort by Assignee most. It is majorly disruptive in issue and PR management.

<img width="398" alt="image" src="https://user-images.githubusercontent.com/63486087/217322445-5766117b-6e15-4b7a-88e0-fd3d859d5b3f.png">
